generators/app/templates/7/scripts/wraith_testing.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
For standards compliment, could you name this file _wraith_testing? Then, ensure it get the right name on the generated project.